holmes/plugins/toolsets/grafana/base_grafana_toolset.py

```diff
@@ -33,11 +33,11 @@ def __init__(
 
     def prerequisites_callable(self, config: dict[str, Any]) -> Tuple[bool, str]:
         if not config:
-            logging.debug(f"Grafana config not provided {self.name}")
+            logging.exception(f"Grafana config not provided {self.name}")
```

Comment:
The docs for `logging.exception()` mention it should only be called from an exception handler. Isn't `logging.error` or `logging.warning` more appropriate here?

Reply:
Let me revert this change. config is not set is possible and the check just fails.